src/yarp.c

```diff
@@ -8,6 +8,10 @@ char* yp_version(void) {
   return YP_VERSION_MACRO;
 }
 
+#define DEBUG_MODE 0
```

Comment:
Can you use `ifndef` here? That way we could pass `-D` if we wanted to turn it on (I think?)

Reply:
Changed! But I couldn't figure out the correct make incantation to change the flag. Was trying the suggestions [here](https://stackoverflow.com/questions/54094963/passing-parameters-to-make-to-change-define-variable). Any thoughts?